etl/src/workers/apply.rs

```diff
@@ -122,7 +122,11 @@ where
     async fn start(self) -> Result<ApplyWorkerHandle, Self::Error> {
         info!("starting apply worker");
 
-        let apply_worker_span = tracing::info_span!("apply_worker");
+        let apply_worker_span = tracing::info_span!(
+            "apply_worker",
+            pipeline_id = self.pipeline_id,
+            publication_name = self.config.publication_name
```

Comment:
Are `pipeline_id` and `publication_name` useful for every log line within the span? There's going to only one of each while the replicator is running. As opposed to something like a `table_id` which will be different.

Reply:
My idea was that we want a reliable way to get a view of all logs of a certain pipeline for debugging a single customer.